lib/Jsrt/Jsrt.cpp

```diff
@@ -957,7 +957,7 @@ CHAKRA_API JsBooleanToBool(_In_ JsValueRef value, _Out_ bool *boolValue)
 
 CHAKRA_API JsConvertValueToBoolean(_In_ JsValueRef value, _Out_ JsValueRef *result)
 {
-    return ContextAPIWrapper<JSRT_MAYBE_TRUE>([&] (Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
+    return ContextAPIWrapper<JSRT_MAYBE_TRUE, JSRT_MAYBE_TRUE>([&] (Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
```

Comment:
> Assumption; Js land calls to a user defined function. This function reaches multiple has/get/set calls before finishing its' 

We know for sure, these are called inside a user defined function and that function will collect on exit. I made them `JSRT_MAYBE_TRUE` in order to keep ChakraFull intact during the process. If we are happy with the result, We might as well remove it? I will add a TODO note there!